src/pages/resultsView/mutation/Mutations.tsx

```diff
@@ -73,24 +79,31 @@ export default class Mutations extends React.Component<
                         tabButtonStyle="pills"
                         unmountOnHide={true}
                     >
-                        {this.generateTabs(this.props.store.hugoGeneSymbols!)}
+                        {this.generateTabs(
+                            this.props.store.hugoGeneSymbols,
+                            this.props.store.mutationsByGene.result
+                        )}
                     </MSKTabs>
                 )}
             </div>
         );
     }
 
-    protected generateTabs(genes: string[]) {
+    protected generateTabs(
+        genes: string[],
+        mutationsByGene: {
+            [hugoGeneSymbol: string]: Mutation[];
+        }
+    ) {
         const tabs: JSX.Element[] = [];
 
         genes.forEach((gene: string) => {
-            const mutationMapperStore = this.props.store.getMutationMapperStore(
-                gene
-            );
+            const mutations = mutationsByGene
```

Comment:
if it's true that mutationsByGene is sometimes undefined, then it should probably be listed as an optional argument above or |undefined, so that ts will enforce this